src/Standards/Squiz/Docs/Commenting/BlockCommentStandard.xml

```diff
@@ -0,0 +1,316 @@
+        <code title="Invalid: Uses multiple //.">
+        <![CDATA[
+<em>//</em> A block comment
+<em>//</em> with
+<em>//</em> multiple lines.
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: Uses /* */.">
+        <![CDATA[
+/*
+    A block comment
+    with
+    multiple lines.
+*/
+        ]]>
+        </code>
+        <code title="Invalid: Uses multiple #.">
+        <![CDATA[
+<em>#</em> A block comment
+<em>#</em> with
+<em>#</em> multiple lines.
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    If asterisks are not used, the contents should be indented by at least 4 spaces.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Indented by at least 4 spaces.">
+        <![CDATA[
+/*
+    A block comment
+      with
+        multiple lines.
+*/
+        ]]>
+        </code>
+        <code title="Invalid: Indented by less than 4 spaces.">
+        <![CDATA[
+/*
+<em> </em>A block comment
+<em>  </em>with
+<em>   </em>multiple lines.
+*/
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    If asterisks are used, they should be aligned.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Asterisks are aligned.">
+        <![CDATA[
+/*
+ * A block comment
+ * with
+ * multiple lines.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Asterisks are not aligned.">
+        <![CDATA[
+/*
+ * A block comment
+  <em>* with</em>
+ * multiple lines.
+<em>*/</em>
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    A block comment should not be empty.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: A block comment with contents.">
+        <![CDATA[
+/*
+    A block comment.
+*/
+        ]]>
+        </code>
+        <code title="Invalid: An empty block comment.">
+        <![CDATA[
+/*
+<em></em>
+*/
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Single line block comments are not allowed.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Uses multiple lines.">
+        <![CDATA[
+/*
+    A block comment.
+*/
+        ]]>
+        </code>
+        <code title="Invalid: Uses a single line.">
+        <![CDATA[
+<em>/* A block comment. */</em>
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Block comment text should start on a new line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Text starts on a new line.">
+        <![CDATA[
+/*
+    A block comment.
+*/
+        ]]>
+        </code>
+        <code title="Invalid: Text starts on the same line.">
+        <![CDATA[
+/* <em>A block comment.</em>
+*/
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The block comment closer should be on a new line.
```

Comment:
I haven't checked, but should it just be on a new line or should it be on a line by itself ? (i.e. with nothing after it either)

Reply:
For example:
```php
<?php
/*
 * A block comment
 * with multiple lines.
 *
 * And a second paragraph.
 */doSomething();
```
This triggers `Squiz.Commenting.BlockComment.NoEmptyLineAfter`.

Do you think an example of this should be included in a code comparison here, or should I add a second example (in the same code comparison block) [here](https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/c0747c7596cf50ddc618d916fe7cbbfc07c91b88/src/Standards/Squiz/Docs/Commenting/BlockCommentStandard.xml#L258)?